front/gatsby/src/components/Login.js

```diff
@@ -17,20 +17,69 @@ const mapDispatchToProps = dispatch => {
     }
 }
 
-const ConnectedLogin = ({logedIn}) => {
+const ConnectedLogin = ({ logedIn, login }) => {
     const isBrowser = typeof window !== 'undefined';
 
     if(isBrowser && logedIn){
         navigate('/articles')
     }
 
+    const usernameInput = useRef(null)
+    const passwordInput = useRef(null)
+
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        const username = usernameInput.current.value
+        const password = passwordInput.current.value
+
+        fetch(env.BACKEND_ENDPOINT + '/login', {
+            method: "POST",
+            // this parameter enables the cookie directive (set-cookie)
+            credentials: 'include',
+            headers: {
+                "Content-Type": "application/json",
+                "Accept": "application/json",
+            },
+            body: JSON.stringify({ username, password })
+        })
+        .then(response => {
+            return response.ok ? response.json() : Promise.reject(new Error('Email or password is incorrect'))
+        })
+        .then(login)
+        .catch(error => {
+            console.error(error)
+            alert(error)
+        })
+
+    }
+
     return (<>
         <section className={styles.disclaimer}>
           <p>Looking for technical and editing support?<br/>Join the <a href="https://ecrituresnumeriques.ca/en/2019/10/25/Stylo-technical-and-editing-support" target="_blank">weekly session</a> for Stylo users.</p>
         </section>
 
         <section className={styles.box}>
             <h1>Login</h1>
+            
+            
+            
+            <form onSubmit={handleSubmit}>

```

Comment:
@antoinentl ça sera surement à améliorer en terme d'affichage

Reply:
(Au passage, j'ai corrigé la typo "Human-num" -> "Huma-Num")

Reply:
À valider par @antoinentl - peut-être aussi demander à Stéphane?
Je dirais:

Connect via Huma-Num (recommended)
Connect with Huma-Num (Boutton)
Create a Huma-Num account (Boutton)

Connect with an existing Stylo account.

Et je rajouterais un disclaimer:

If you use the same email address for your Stylo account and for your Huma-Num account, the two accounts will be automatically merged.